index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <title>
+    Document
+  </title>
+</head>
+
+<body>
+  <div class=border>
+    <h1> Jaye Lin Resume </h1>
+    <div class="contact">
+      <p> <a href="Illusxia. carbonmade.com"> Portfolio </a><br>
+
+        <a href="Xiaoxxia23@Gmail.com"> Email </a> <br>
+        Baltimore, MD
+      </p>
+    </div>
+
+    <div class="edu">
+      <h3> Education And Training: </h3>
+      <p>
+        <b>Maryland Institute Of Art And Design</b>
+        <br />
+        <i>(2019-Current)</i><br />
+        Bachelor of graphic design
+      </p>
+
+      <p>
+        <b>University Of Minnesota- Twin Cities</b><br />
+        <i>(2014-2018)</i><br />
+        Bachelors of fine arts<br />
+        Deans list
+      </p>
+
+      <p>
+        <b> Animation mentor</b><br />
+        <i>(2018)</i><br />
+        Character animation
+      </p>
+    </div>
+
+    <div class="expert">
```

Comment:
Same note about using `section` over `div` here and the other `div`s that are immediately followed by a heading element. That's usually a sure sign that you could and should use a `section` 👍🏻